helion/_compiler/inductor_lowering.py

```diff
@@ -59,6 +61,26 @@
     CodegenHandler = Callable[["GraphInterpreter", torch.fx.Node], object]
 
 
+@contextlib.contextmanager
+def patch_inductor_lowerings() -> Generator[  # pyre-ignore[3]
+    None, Any, Any
+]:
+    """Context manager to temporarily patch the inductor lowering table.
+
+    This is useful for modifying specific Inductor lowerings without
+    affecting the global state, especially in cases where Helion
+    is missing support for a specific lowering.
+    """
+    original_lowerings = torch._inductor.lowering.lowerings.copy()
+    try:
+        from helion._compiler import inductor_lowering_extra
```

Comment:
The second time this is run it will do nothing, since imports are cached in Python.  So I don't think this will work more than once.